src/libraries/System.Net.NetworkInformation/tests/FunctionalTests/IPGlobalPropertiesTest.cs

```diff
@@ -118,6 +122,7 @@ public void IPGlobalProperties_TcpActiveConnections_NotListening()
         }
 
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsSubsystemForLinux))] // [ActiveIssue("https://github.com/dotnet/runtime/issues/18258")]
+        [PlatformSpecific(~TestPlatforms.Browser)] // throws EntryPointNotFoundException : SystemNative_EnumerateInterfaceAddresses
```

Comment:
What would cause it to not find that entrypoint?

Reply:
I filed the issue: https://github.com/dotnet/runtime/issues/38987